opm/autodiff/BlackoilModelBase_impl.hpp

```diff
@@ -195,9 +194,10 @@ namespace detail {
                 // Only rank 0 does print to std::cout if terminal_output is enabled
                 terminal_output_ = (info.communicator().rank()==0);
             }
-            int local_number_of_wells = wells_ ? wells_->number_of_wells : 0;
+            int local_number_of_wells = localWellsActive() ? wells().number_of_wells : 0;
             int global_number_of_wells = info.communicator().sum(local_number_of_wells);
-            wells_active_ = ( wells_ && global_number_of_wells > 0 );
+            const bool wells_active = ( localWellsActive() && global_number_of_wells > 0 );
```

Comment:
You are right, thanks for pointing it out. It will probably cause problem for parallel running later, while I have not made testing for parallel running as regular. 

Updated.